doc/classes/RichTextLabel.xml

```diff
@@ -112,6 +112,15 @@
 				[b]Note:[/b] If [member threaded] is enabled, this method returns a value for the loaded part of the document. Use [method is_finished] or [signal finished] to determine whether document is fully loaded.
 			</description>
 		</method>
+		<method name="get_line_range">
+			<return type="Vector2i" />
+			<param index="0" name="line" type="int" />
+			<description>
+				Returns the range of characters of the line.
```

Comment:
Please be more explicit here. Going by this description alone I still would not know what this method returns.

Reply:
I admittedly just copied from `TextParagraph.get_line_range()`, but you're absolutely right. Would something like this be good?
`Returns the first and last visible character index for the line index provided.`
Or do you have something else in mind?

Reply:
Thank you, I can work with that
```suggestion
				Returns the indexes of the first and last visible characters for the given [param line], as a [Vector2i].
```